txscript/standard_test.go

```diff
@@ -1013,3 +1013,28 @@ func TestStringifyClass(t *testing.T) {
 		}
 	}
 }
+
+// TestNullDataScript tests whether NullDataScript returns a valid script.
+func TestNullDataScript(t *testing.T) {
+	data := []byte{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,
```

Comment:
Great, well that would explain why my original function was wrong because I just followed the existing isNullData function to make something that would fit it. I have updated my pr, and one of the tests doesn't pass yet because of this bug. It will pass once isNullData is fixed though.